build.gradle.kts

```diff
@@ -184,17 +196,21 @@ subprojects {
                     targetsContainer.register("api22") {
                         deviceName = "API22"
 
-                        scheduling = com.avito.instrumentation.configuration.target.scheduling.SchedulingConfiguration()
-                            .apply {
-                            quota = com.avito.instrumentation.configuration.target.scheduling.quota.QuotaConfiguration()
-                                .apply {
+                        scheduling = SchedulingConfiguration().apply {
+                            quota = QuotaConfiguration().apply {
                                 retryCount = 1
                                 minimumSuccessCount = 1
                             }
 
-                            reservation = com.avito.instrumentation.configuration.target.scheduling.reservation.TestsBasedDevicesReservationConfiguration()
-                                .apply {
-                                device = com.avito.instrumentation.reservation.request.Device.Emulator.Emulator22
+                            reservation = TestsBasedDevicesReservationConfiguration().apply {
+                                device = CloudEmulator(
+                                    name = "api22",
+                                    api = 22,
+                                    model = "Android_SDK_built_for_x86",
+                                    image = "avitotech/android-emulator-22:24e6ed0ec4",
+                                    cpuCoresRequest = "1",
+                                    cpuCoresLimit = "1.3"
+                                )
```

Comment:
How do we pass credentials for the cloud?

Reply:
dockerhub is available without authentication